src/ir/datalog/attribute.h

```diff
@@ -0,0 +1,61 @@
+//-----------------------------------------------------------------------------
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//----------------------------------------------------------------------------
+
+#ifndef SRC_IR_DATALOG_ATTRIBUTE_H_
+#define SRC_IR_DATALOG_ATTRIBUTE_H_
+
+#include <memory>
+
+#include "src/ir/datalog/value.h"
+
+namespace raksha::ir::datalog {
+
+constexpr char kStringAttributePayloadName[] = "StringAttributePayload";
```

Comment:
`inline constexpr` here and below. (https://abseil.io/tips/140 for context)

Reply:
Done, thanks for the tip.